### ISSUE TITLE
[Wanted - Build Contest] Some fancy building

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#1847]
Okay so now that we have terrain in latest 20w17d, I would like to share some nice building and use some of it for a starting zone.
I will ask something a bit more specific for the starting zone (tutorial, ect), so for now it is just for ambiance!
You can use inspiration from some of our concept art, or do something that you like.

Thanks!

![StationBuildings01](https://user-images.githubusercontent.com/6604318/80273679-6a8b6180-870f-11ea-9c27-b770a95c303c.jpg)

**Comment by tsunamayo** [comment]
Ps - if you start working on this, please share some WIP! I might as well show some wip on some social media, it dont have to be 100% perfect... It is a bit hard for me to announce a feature without anything to show...

**Comment by Aiwii** [comment]
I shall attempt to build something

**Comment by alxnns1** [comment]
![20200425141310_1](https://user-images.githubusercontent.com/485571/80281008-fd240480-86ff-11ea-89a0-3a5f4e193a46.jpg)
Should we post WIP screenshots here?

**Comment by Xerma** [comment]
I know the image states for there to be a 4m margin around the building but would it be fine for there to be small decorations around the base so long as they don't stick out? I also want to make sure that it's alright if we have the upper sections of building extend a little bit out as well as it can add some variety in the contrast of buildings. Obviously we won't want them to stick out super far.

WIP Shop Building Thing:
![20200425153811_1](https://user-images.githubusercontent.com/32974671/80290521-21391300-870b-11ea-9712-9e9190cd6177.jpg)

**Comment by ZachZent** [comment]
![](https://cdn.discordapp.com/attachments/702281585138991175/703819110710247434/unknown.png)

Military base/landing zone

**Comment by tsunamayo** [comment]
Whoa some cool stuff over here! Yes if you could paint it - the way you want it is not important for now. @Xerma yes you dont have to respect this 100%, it was just guide really!
Thanks a lot all!

**Comment by alxnns1** [comment]
![20200425175703_1](https://user-images.githubusercontent.com/485571/80306024-8ac32b00-87b8-11ea-9580-ad333a870a6b.jpg)
A nice copper and dark red colour scheme, how important are interiors? Should they be fully detailed or are simple rooms enough?

**Comment by ExodistSKY1** [comment]
![20200426015802_1](https://user-images.githubusercontent.com/45324484/80325510-1493e800-87fb-11ea-90d3-ce351838e519.jpg)
![20200426023306_1](https://user-images.githubusercontent.com/45324484/80325512-152c7e80-87fb-11ea-9844-63a896491d28.jpg)
![20200426142020_1](https://user-images.githubusercontent.com/45324484/80325513-152c7e80-87fb-11ea-91c7-1fe83e755271.jpg)
![20200426150033_1](https://user-images.githubusercontent.com/45324484/80325514-15c51500-87fb-11ea-846e-e4b249c4022d.jpg)
![20200426151944_1](https://user-images.githubusercontent.com/45324484/80325515-15c51500-87fb-11ea-8b57-7811a0a02c81.jpg)
![20200426181835_1](https://user-images.githubusercontent.com/45324484/80325516-15c51500-87fb-11ea-9af9-f13560d5febe.jpg)
![20200426191152_1](https://user-images.githubusercontent.com/45324484/80325517-165dab80-87fb-11ea-884e-a2593353e473.jpg)
![20200426194232_1](https://user-images.githubusercontent.com/45324484/80325518-165dab80-87fb-11ea-8483-d8b3b7f88572.jpg)

**Comment by tsunamayo** [comment]
Okay pretty cool build everybody! @alxnns1 no no detailed interior, but some clean looking room with some space. I might want to put some tutorial inside! Thanks

**Comment by ultrak2k** [comment]
![20200428200427_1](https://user-images.githubusercontent.com/43084360/80463206-fe696180-898b-11ea-903f-3641daf1e916.jpg)
![20200428172858_1](https://user-images.githubusercontent.com/43084360/80463211-00332500-898c-11ea-853e-5577d89a910b.jpg)
A factory-industrial building I'm making, gonna be big
(WIP)

**Comment by Citronix** [comment]
A small (20x21x20) gas station
![711980_20200427171223_1](https://user-images.githubusercontent.com/33103748/80466863-5fb61400-893d-11ea-9bcf-4bd3ade81c9c.png)
![711980_20200428102732_1](https://user-images.githubusercontent.com/33103748/80466881-63e23180-893d-11ea-8d1a-4bf728d87af7.png)
![711980_20200427161404_1](https://user-images.githubusercontent.com/33103748/80466911-6ba1d600-893d-11ea-9a04-b206ffbe9d99.png)

**Comment by Majikmonster** [comment]
A couple city-themed apartments, both using an 8x8 footprint on a 16x16 base.
![20200428114940_1](https://user-images.githubusercontent.com/51390907/80509658-b6c0e680-8947-11ea-9f29-30caf4bd840c.jpg)
![20200428115807_1](https://user-images.githubusercontent.com/51390907/80509672-b9bbd700-8947-11ea-82ae-2732458af54f.jpg)
Works in progress. Minimal interior layout.

**Comment by YnotnA** [comment]
WIP : Interior and details
Maybe a store

![image](https://user-images.githubusercontent.com/24494987/80513838-f1467580-897f-11ea-9924-fb6688bedfc5.png)
![image](https://user-images.githubusercontent.com/24494987/80514096-54d0a300-8980-11ea-976f-d7835d9dc244.png)
![image](https://user-images.githubusercontent.com/24494987/80513862-fb687400-897f-11ea-8147-f616c51658c6.png)
![image](https://user-images.githubusercontent.com/24494987/80514147-69ad3680-8980-11ea-9065-5265dd84ab16.png)
![image](https://user-images.githubusercontent.com/24494987/80520099-3cb15180-8989-11ea-847a-82afc1b931ab.png)
![image](https://user-images.githubusercontent.com/24494987/80519991-1ee3ec80-8989-11ea-8dbd-5b6cabacb160.png)
![image](https://user-images.githubusercontent.com/24494987/80519878-f4922f00-8988-11ea-8cea-c2fea2dc9a7c.png)

Idea maybe a bar :
![image](https://user-images.githubusercontent.com/24494987/80519381-31115b00-8988-11ea-99f3-2a8776f33356.png)

**Comment by Citronix** [comment]
WIP : Housing apartement (curently only one bedroom)
![711980_20200428203121_1](https://user-images.githubusercontent.com/33103748/80524664-45f1ec80-8990-11ea-8a85-67153dab58a6.png)
Outside view
![711980_20200428202513_1](https://user-images.githubusercontent.com/33103748/80524682-4ee2be00-8990-11ea-8b83-30e8e3f219fc.png)
2nd Floor Bedroom
![711980_20200428195942_1](https://user-images.githubusercontent.com/33103748/80524715-5bffad00-8990-11ea-9f5a-fceffd968dcc.png)
Bed
![711980_20200428201522_1](https://user-images.githubusercontent.com/33103748/80524724-5f933400-8990-11ea-8e8c-4ccc55a6d63a.png)
Computer
![711980_20200428172241_1](https://user-images.githubusercontent.com/33103748/80524754-6a4dc900-8990-11ea-8c04-dc8a723c2483.png)
1rst Floor

**Comment by tsunamayo** [comment]
okay very cool all! @alxnns1 @YnotnA @Xerma @Citronix @ZachZent @ultrak2k @Aiwii @ExodistSKY1 
if you could give me some blueprint, so I can make a post of the steam page - and also do a devlog! Thanks

**Comment by YnotnA** [comment]
@tsunamayo 
[Starniverse.zip](https://github.com/tsunamayo/Starship-EVO/files/4556679/Starniverse.zip)

**Comment by Citronix** [comment]
Here the finished gas station. I'm still working on the house/hotel.
[Gas Station mk1.zip](https://github.com/tsunamayo/Starship-EVO/files/4557041/Gas.Station.mk1.zip)

**Comment by ExodistSKY1** [comment]
The Energy harvester/Reactor I have been working on. 
[Power Reactor.zip](https://github.com/tsunamayo/Starship-EVO/files/4557257/Power.Reactor.zip)

**Comment by alxnns1** [comment]
My country house
[Country House.zip](https://github.com/tsunamayo/Starship-EVO/files/4557871/Country.House.zip)

**Comment by Aiwii** [comment]
My two buildings
[BuildingOne.zip](https://github.com/tsunamayo/Starship-EVO/files/4560309/BuildingOne.zip)
[Building Two.zip](https://github.com/tsunamayo/Starship-EVO/files/4560311/Building.Two.zip)

**Comment by hl2shadowslayer** [comment]
Here's the military fortress the Evo Network community has been working on. (Had to upload it on Zach's behalf because he's having github issues)
[Fortress.zip](https://github.com/tsunamayo/Starship-EVO/files/4561633/Fortress.zip)

**Comment by Majikmonster** [comment]
Does this mean the build contest is over? It isn't entirely clear what marks the end of it from the initial post.

**Comment by Womble-TC** [comment]
Yeah, I'm also working on something, are you still accepting submissions?

**Comment by tsunamayo** [comment]
Hello, thanks to all! Nothing is really over, you can still continue to submit build! @Womble-TC okay if you would like to build a building (...), what could be of great use to me is a tutorial room building, very much like the old one if you recal it! No heavy detailling, wall / floor use hull block. Cheers

**Comment by Womble-TC** [comment]
yup, I remember the old one, I should be able to make something similar.

**Comment by capncereal** [comment]
![image](https://user-images.githubusercontent.com/15242472/82103083-5962be80-96d7-11ea-9b7a-2f6c107d25a9.png)

Apartment/hotel style building WIP. 

mostly practical/compact studio-apartment style dwellings 
![image](https://user-images.githubusercontent.com/15242472/82103118-81522200-96d7-11ea-821a-29232f65597f.png)

windows frosted on the lower levels for some privacy 
![image](https://user-images.githubusercontent.com/15242472/82103155-9cbd2d00-96d7-11ea-8d6c-f4936dea0154.png)

simple full-bathroom 
![image](https://user-images.githubusercontent.com/15242472/82103164-a9da1c00-96d7-11ea-95de-7f0551be2c75.png)

**Comment by Servomotor** [comment]
HOLY COW...there are some amazing works here. Makes me feel embarrassed with my sub. 
[Servo Country House 20w21c.zip](https://github.com/tsunamayo/Starship-EVO/files/4679045/Servo.Country.House.20w21c.zip)

![Country House](https://user-images.githubusercontent.com/65745916/82846173-342b3880-9ead-11ea-947a-1ddf621d26ae.png)

**Comment by tsunamayo** [comment]
@Servomotor very very sweet build and detailling! @capncereal looking good so far!
